src/foam/nanos/menu/DAOMenu.js

```diff
@@ -133,9 +142,14 @@ foam.CLASS({
         throw new Error('No DAO found for ' + this.daoKey);
       }
 
+      var dao = X[this.daoKey];
+      dao.of = this.of ? this.of : dao.of;
+
+      var predicatedDAO = dao.where(this.predicate);
```

Comment:
What if `predicate` isn't set? Shouldn't this be something like
```suggestion
      var predicatedDAO = this.predicate ? dao.where(this.predicate) : dao;
```